mappings/net/minecraft/client/render/BuiltChunkStorage.mapping

```diff
@@ -0,0 +1,29 @@
+CLASS net/minecraft/class_769 net/minecraft/client/render/BuiltChunkStorage
+	FIELD field_4146 worldRenderer Lnet/minecraft/class_761;
+	FIELD field_4147 sizeZ I
```

Comment:
I think xSize xOffset etc fit our "natural order" naming style better, sizeX etc is prefix style

Reply:
Right now we're naming coordinates the other way (see [conventions](https://github.com/FabricMC/yarn/blob/19w44a/CONVENTIONS.md#coordinates), and also almost all current coordinate fields are named this way). The idea is that the fields is the size['s] x component.

The other order could work too, but I think it would be better to make a separate issue to change the conventions as well as rename fields currently named this way.

Reply:
Right, I didn't even realize we added this to the conventions and I agree this should be fixed separately if desired.

For further discussion in this area it should be noted that the order isn't fixed with a natural scheme, it depends on subject ordering. There would be e.g. xSize and minX. The other form sizeX refers to a size called "x" which makes little sense and xMin implies that the x axis itself had a minimum, which it doesn't.